Added catalan ca_ES language translation file

caos30 opened the pull request:
I only added the file: /src/lang/ca_ES.lang.php
I'm using it on my production server without problems ;-)
Cheers!
Sergi

caos30 committed:
Added catalan ca_ES language translation file

caos30 committed:
Merge pull request #1 from caos30/caos30-catalan-lang

Added catalan ca_ES language translation file

TimZ99 commented on src/lang/ca_ES.lang.php:
```diff
@@ -0,0 +1,327 @@
+		// date/time format according the strftime php function format parameter http://php.net/manual/function.strftime.php
+		'short_day_format' => '%e %B',
+		'long_day_format' => '%e %B %Y',
+		'yesterday_format' => 'Yesterday at %k:%M',
```
```suggestion
		'yesterday_format' => 'Ahir a %k:%M',
```

TimZ99 commented on src/lang/ca_ES.lang.php:
```diff
@@ -0,0 +1,327 @@
+		'short_day_format' => '%e %B',
+		'long_day_format' => '%e %B %Y',
+		'yesterday_format' => 'Yesterday at %k:%M',
+		'other_day_format' => '%A at %k:%M',
```
```suggestion
		'other_day_format' => '%A a les %k:%M',
```

TimZ99 commented on src/lang/ca_ES.lang.php:
```diff
@@ -0,0 +1,327 @@
+		'server' => 'Servidors',
+		'server_log' => 'Log',
+		'server_status' => 'Estat',
+		'server_update' => 'FER PING ARA',
```
```suggestion
		'server_update' => 'Actualització',
```

caos30 replied:
Believe me, i invested more than 2 hours thinking about the label for this button. "Actualització" (literally means Update as NOUN, not as verb) is not a good translation. 

In fact, this button is special in it behaviour. It does a "ping" (or a "request") to all the defined servers/websites defined in the system. So it is very difficult to find an adequate expression in a single word. If you ask me: "update" is a very bad choose, because it means a lot of different things depending on the context. So in this case, as a "button" in the main menu of the webapp... it says nothing. Even for me that i'm using since sveral months the app, sometimes confus to me this "update" label!!!

So, i really choose the concept of "TO DO A PING NOW" as a very much better expression of the consequences of do click over that button. In catalan, this expression is just very brief: "fer ping ara".

I choosed to put in CAPITAL LETTERS to be able to differenciate the 3 words as a unique "clickable entity". Because if you put it in lowercase then they seem 3 different option menu. If you ask me: this is a bad design of the HTML/CSS of the menu. Take in account that other menu options in some languages can have 2 or more words to express the same that in english only need one. So, i would design that menu taking this in account.

TimZ99 commented on src/lang/ca_ES.lang.php:
```diff
@@ -0,0 +1,327 @@
+		'alert_type_always' => 'Sempre',
+		'alert_proxy' => "Encara que s'habiliti, el proxy no és emprat mai per als serveis",
+		'alert_proxy_url' => '<b>Format:</b> Servidor:Port',
+		'log_status' => 'LOG d\'estat',
```
```suggestion
		'log_status' => 'Log d\'estat',
```

caos30 replied:
As i commented in other comment of this pull request, it's necessary to put LOG in CAPITAL LETTERS because is a FOREIGN WORD (english) in a catalan phrase, and it's necessary to differentiate or mark it in some way. Uppercase it's a great and simple way. Just like you write USA, although the correct way would be U.S.A.

TimZ99 commented on src/lang/ca_ES.lang.php:
```diff
@@ -0,0 +1,327 @@
+		'alert_proxy' => "Encara que s'habiliti, el proxy no és emprat mai per als serveis",
+		'alert_proxy_url' => '<b>Format:</b> Servidor:Port',
+		'log_status' => 'LOG d\'estat',
+		'log_status_description' => "Si el LOG d'estat es configura a SÍ, l'aplicació enregistrarà aquells events que disparin les notificacions.",
```
```suggestion
		'log_status_description' => "Si el log d'estat es configura a SÍ, l'aplicació enregistrarà aquells events que disparin les notificacions.",
```

TimZ99 commented on src/lang/ca_ES.lang.php:
```diff
@@ -0,0 +1,327 @@
+		'email_smtp_noauth' => 'Deixar en blanc si no cal autenticació',
+		'sms_status' => 'Permetre l\'enviament de missatges de text',
+		'sms_gateway' => 'Servei \'Gateway\' per l\'enviament de SMS',
+		'sms_gateway_mosms' => 'Mosms',
```
sms_gateway_.... are no longer in use. Please update to latest version. You can use [this](https://github.com/TimZ99/phpservermon-language-editor) tool if you want.

caos30 replied:
Well, the problem is not my tool used. I realize that my error was to use the master branch as reference. I should take the develop branch. Let me download that develop version and i will do a new commit and pull request.

I apologise by waste your time... i'm very sorry :-(
Thanks anyway for your patient.
Sergi

TimZ99 replied:
No problem at all! It’s worth it 👍

TimZ99 commented on src/lang/ca_ES.lang.php:
```diff
@@ -0,0 +1,327 @@
+		'pushover_error' => "Ha succeït un error provant d'enviar la notificació Pushover: %s",
+		'pushover_error_noapp' => "No ha estat posible enviar la notificació de prova: no s'ha trobat cap token d'API d'una App Pushover a la configuració general.",
+		'pushover_error_nokey' => "No ha estat posible enviar la notificació de prova: no s'ha trobat cap clau Pushover al vostre perfil d'usuari.",
+		'log_retention_period' => "Període de retenció al LOG",
```
```suggestion
		'log_retention_period' => "Període de retenció al log",
```

caos30 replied:
"log" is an english word. So it cannot be used inside another language paragraph without some kind of special mark. It could be using an _italic_ font-style, or "double quoting" it. But because this label translation could be used in unknown circumstances (for example an javascript native alert() function cannot use italic or htmlentities), it's safe to put LOG using capital letters, to differenciate this word from the other catalan words.

It applies to other "foreign" languages (other than english) when inserting english words.

TimZ99 commented on src/lang/ca_ES.lang.php:
```diff
@@ -0,0 +1,327 @@
+	),
+	// for newlines in the email messages use <br/>
+	'notifications' => array(
+		'off_sms' => 'El servidor \'%LABEL%\' NO RESPON: IP=%IP%, Port=%PORT%. Error=%ERROR%',
```
Notifications have a new variable. Please update to the latest version.

TimZ99 commented on src/lang/ca_ES.lang.php:
```diff
@@ -0,0 +1,327 @@
+ **/
+
+$sm_lang = array(
+	'name' => 'Català',
```
```suggestion
	'name' => 'Català - Catalán',
```

TimZ99 commented on src/lang/ca_ES.lang.php:
```diff
@@ -0,0 +1,327 @@
+		'no' => 'No',
+		'insert' => 'Inserir',
+		'add_new' => 'Afegir nou',
+		'update_available' => 'Hi ha disponible una nova versió ({version}) a <a href="http://www.phpservermonitor.org" target="_blank">http://www.phpservermonitor.org</a>.',
```
URL has been changed. See the latest update.

TimZ99 reviewed (changes requested):
Thanks for the translation!
It needs to be updated to the latest version (develop branch) as there are some 'breaking' changes.

caos30 committed:
Updated the catalan translation ca_ES.lang.php 

Updated the catalan translation ca_ES.lang.php taking as reference difference with last version of english translation file.

caos30 committed:
Quit sms_gateway labels not used on catalan

Removed about 10 translation labels with prefix sms_gateway_ on catalan translation file (ca_ES.lang.php) not used any more. I think that it really is the definitive good version ;-)

TimZ99 committed:
Changed url

TimZ99 committed:
Update ca_ES.lang.php

TimZ99 committed:
Added Catalan to translations

TimZ99 commented:
Great work!

caos30 commented:
Thanks to you Tim. It's really a great opensource project. I use it to monitor all my websites.

In fact, i have done a DARK THEME ;-) see... the image below. But i'm waiting for to have a little of time to put it "choosable" for the user. I mean... now i only have made a rude hacking of the original theme. I want to add to the interface at least an icon to toggle between light/dark theme ;-)

Do you think that this feature can be included in the main project if i develop it?

Best regards!

![web-monitor-dark-theme](https://user-images.githubusercontent.com/1767547/65070492-9193e080-d941-11e9-8350-b193a0ab06fd.png)
